okio-wasifilesystem/src/wasmMain/kotlin/okio/internal/preview1/Preview1.kt

```diff
@@ -130,6 +130,51 @@ internal external fun path_open(
   returnPointer: PointerU8,
 ): Int // should be Short??
 
+/**
+ * path_rename(fd: fd, old_path: string, new_fd: fd, new_path: string) -> Result<(), errno>
+ *
+ * Rename a file or directory.
+ * Note: This is similar to `renameat` in POSIX.
+ */
+@WasmImport("wasi_snapshot_preview1", "path_rename")
+internal external fun path_rename(
+  fd: fd,
+  old_path: PointerU8,
+  old_pathSize: size,
+  new_fd: fd,
+  new_path: PointerU8,
+  new_pathSize: size,
+): Int // should be Short??
+
+/**
+ * path_symlink(old_path: string, fd: fd, new_path: string) -> Result<(), errno>
+ *
+ * Create a symbolic link.
+ * Note: This is similar to `symlinkat` in POSIX.
+ */
+@WasmImport("wasi_snapshot_preview1", "path_symlink")
+internal external fun path_symlink(
+  old_path: PointerU8,
```

Comment:
The names old_path and new_path look like a copy-paste mistake in the spec. These words confuse me when thinking of symlinks.